### ISSUE TITLE
Linear integration (again) not working in EU

### ISSUE THREAD

**Issue by mfn** [getsentry/sentry#70497]
### Environment

SaaS (https://sentry.io/)

### Steps to Reproduce

This is a follow up to https://github.com/getsentry/sentry/issues/68970

At first it seemed it worked: we could, directly from within sentry, create and link linear issues.

However, then we noticed that the integration linear->slack does not work: linear did not link back to sentry (they've their own space in an issue to show such an integration-link).

Therefore we did reset our sentry integration in linear, because we saw this there:
![image](https://github.com/getsentry/sentry/assets/87493/1d7c7161-a0a8-40a8-8554-9bc57475b994)

And according to there, it worked:
![image](https://github.com/getsentry/sentry/assets/87493/66cedf85-eb91-4452-ad88-0eb27c3b99bd)

But now in sentry, it does not, we only see:
![image](https://github.com/getsentry/sentry/assets/87493/76462196-556f-4873-ac84-8bba92541d1b)

### Expected Result

should work

### Actual Result

does not work

### Product Area

Settings - Integrations

**Comment by mrduncan** [comment]
Hi @mfn, thanks for raising this - we're able to reproduce your scenario and are investigating.

**Comment by GabeVillalobos** [comment]
Posting a brief update on this:

- We're able to consistently reproduce the connection issues you're seeing by disabling and reconnecting a Linear integration on a Sentry organization in the EU region.

- We will continue to provide updates as we learn more and hope to have a fix for this soon.

**Comment by skbogner** [comment]
@GabeVillalobos any chance https://github.com/getsentry/sentry/issues/71025 could have solved anything here?

**Comment by GabeVillalobos** [comment]
@skbogner This does appear to have resolved the problem. 
We also had an outage with external integrations as noted on our [status page](https://status.sentry.io/incidents/g6mvlrtcm92z) that delayed our testing, but this has been resolved.

I retested the installation and issue linking flows and everything appears to be working as intended, so I'll be closing this issue. Feel free to to re-open it if you encounter any further problems with the Linear integration.
